README.md

```diff
@@ -11,116 +11,30 @@ Capabilities:
 
 ## Install and Configure Prerequisites on the Cluster
 
-You'll need a DCOS cluster with one public node and at least five private nodes, DCOS CLI, and DCOS package CLIs.
+You'll need a DC/OS cluster with one public node and at least five private nodes, DC/OS CLI, and DC/OS package CLIs.
 
-Install package CLIs:
+Run `bash stage_ebc_demo.sh [master_ip] [ELB_Hostname]`
 
-```
-$ dcos package install cassandra --cli
-$ dcos package install kafka --cli
-```
+The above command will do the following:
 
-## Demo steps
-
-Install packages for DCOS UI:
-* kafka
-* cassandra
-* zeppelin
-* marathon-lb
-
-Wait until the Kafka and Cassandra services are healthly. You can check their status with:
-
-```
-$ dcos kafka connection
-...
-$ dcos cassandra connection
-...
-```
-
-## Edit the Tweeter Service Config
-
-Edit the `HAPROXY_0_VHOST` label in `tweeter.json` to match your public ELB hostname. Be sure to remove the leading `http://` and the trailing `/` For example:
-
-```json
-{
-  "labels": {
-    "HAPROXY_0_VHOST": "brenden-7-publicsl-1dnroe89snjkq-221614774.us-west-2.elb.amazonaws.com"
-  }
-}
-```
-
-## Run the Tweeter Service
-
-Launch three instances of Tweeter on Marathon using the config file in this repo:
-
-```
-$ dcos marathon app add tweeter.json
-```
-
-The service talks to Cassandra via `node-0.cassandra.mesos:9042`, and Kafka via `broker-0.kafka.mesos:9557` in this example.
-
-Traffic is routed to the service via marathon-lb. Navigate to `http://<public_elb>` to see the Tweeter UI and post a Tweet.
-
-
-## Post a lot of Tweets
-
-Post a lot of Shakespeare tweets from a file:
-
-``` 
-dcos marathon app add post-tweets.json
-```
-
-This will post more than 100k tweets one by one, so you'll see them coming in steadily when you refresh the page. Take a look at the Networking page on the UI to see the load balancing in action.
-
-
-## Streaming Analytics
-
-Next, we'll do real-time analytics on the stream of tweets coming in from Kafka.
+* Update your Python Env to work with the latest CLI
+* Install a local CLI at ./dcos
```

Comment:
This script core does not update local pyenv nor does it install the base CLI (it does install a few package CLIs)